_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+Each individual might choose completely different goods for indirect exchange depending on what they have available and when they are expecting to need them for exchange, so a convergence between individuals to choose the same good for this purpose is initially not necessary. The convergence would come naturally afterward, once humans have discovered the concept of medium of exchange, and that's when the most salable goods begin to compete for being a generalized medium of exchange (i.e. Money). The game theory concept of [focal point](https://en.wikipedia.org/wiki/Focal_point_(game_theory)) kicks in at this stage reinforcing the demand for the most salable good.
+
+A key concept in this debate is the term of the exchange. For a good to be money it is implicit that it serves well its purpose in the short term, which is also implicit in the words "most salable" within Menger's definition of money "the **most salable** of commodities", which [Carlos Bondone](http://www.carlosbondone.com/en/theory-of-economic-time/economic-theory/development-of-concepts.html) has generalized to "the most salable economic good". If it only serves its purpose in longer terms it might not be money but, it would still be a medium of indirect exchange.
+
+But, in this post, we are interested in the bootstrapping process to achieve that salability.  Let's refer back to the stage of goods being just a medium of indirect exchange. I stated above that I would rely on cereal's utility as food to foresee its future demand because the concept of medium of indirect exchange would still be an unknown abstraction for me. If we fast forward to the context where humans already grasp the concept of medium of exchange, why shouldn't they be able to foresee the monetary demand of any good regardless if it has other uses or not? That is to say: easy to transport, divisible, durable, difficult to fake, etc. This is clearly the case for Szabo's "collectibles" and also Bitcoin's case.  From a historical point of view, I find it rather unlikely that humans used otherwise useless goods as a medium of exchange without first having discovered the medium of exchange concept through other consumptive goods, but it is certainly theoretically possible.
```

Comment:
If we fast forward to a context where humans already conceptually grasp mediums of exchange, why should not they be able to foresee the monetary demand of any good, regardless of its other uses -- that is, its ease of transport, divisibility, durability, and unfalsifiability.

Reply:
Remove extra space after "salability.__Let's refer back..." 
Remove space after "also Bitcoin's case.__From a historical..."